universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -12,11 +13,7 @@
 import services.Path;
 import services.question.exceptions.InvalidQuestionTypeException;
 import services.question.exceptions.UnsupportedQuestionTypeException;
-import services.question.types.AddressQuestionDefinition;
-import services.question.types.NameQuestionDefinition;
-import services.question.types.QuestionDefinition;
-import services.question.types.RepeaterQuestionDefinition;
-import services.question.types.TextQuestionDefinition;
+import services.question.types.*;
```

Comment:
(blocking) no wildcard imports

Reply:
you should be able to change this setting in your IDE settings.

Reply:
Wildcards removed in commit: 1e54af6.  I've also updated my IDE settings